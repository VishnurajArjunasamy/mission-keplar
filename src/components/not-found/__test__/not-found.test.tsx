import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import NotFound from "../not-found";

describe("Not found component", () => {
  it("renders not found component", () => {
    const { asFragment } = render(<NotFound />);

    expect(screen.getByText("YOU LOST?")).toBeInTheDocument();
    expect(screen.getByText("YOU LOST?").parentNode).toHaveClass("notFound");

    expect(asFragment()).toMatchSnapshot();
  });
});
