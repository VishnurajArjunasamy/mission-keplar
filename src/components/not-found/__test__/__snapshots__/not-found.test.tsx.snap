// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found component renders not found component 1`] = `
<DocumentFragment>
  <div
    class="notFound"
  >
    <h1>
      YOU LOST?
    </h1>
  </div>
</DocumentFragment>
`;
